src/framework/components/sound/system.js

```diff
@@ -56,50 +45,62 @@ Component._buildAccessors(SoundComponent.prototype, _schema);
 
 Object.assign(SoundComponentSystem.prototype, {
     initializeComponentData: function (component, data, properties) {
-        properties = ['volume', 'pitch', 'positional', 'refDistance', 'maxDistance', 'rollOffFactor', 'distanceModel', 'slots', 'enabled'];
-        ComponentSystem.prototype.initializeComponentData.call(this, component, data, properties);
+        properties = [
```

Comment:
Is there a reason for changing the order of the properties? Usually the order matters in initialization not sure if this will break something...

Reply:
Restored to original order.